static/app/views/replays/detail/layout.tsx

```diff
@@ -0,0 +1,263 @@
+import BaseResizePanel from 'react-resize-panel';
+import styled from '@emotion/styled';
+
+import ErrorBoundary from 'sentry/components/errorBoundary';
+import ReplayTimeline from 'sentry/components/replays/breadcrumbs/replayTimeline';
+import ReplayView from 'sentry/components/replays/replayView';
+import space from 'sentry/styles/space';
+import useFullscreen from 'sentry/utils/replays/hooks/useFullscreen';
+import theme from 'sentry/utils/theme';
+
+import Breadcrumbs from './breadcrumbs';
+import FocusArea from './focusArea';
+import FocusTabs from './focusTabs';
+
+type Layout =
+  /**
+   * ### Sidebar
+   * ┌───────────────────┐
+   * │ Timeline          │
+   * ├──────────┬────────┤
+   * │ Details  > Video  │
+   * │          >        │
+   * │          >^^^^^^^^┤
+   * │          > Crumbs │
+   * │          >        │
+   * └──────────┴────────┘
+   */
+  | 'sidebar'
+  /**
+   * ### Topbar
+   *┌────────────────────┐
+   *│ Timeline           │
+   *├───────────┬────────┤
+   *│ Video     │ Crumbs │
+   *│           │        │
+   *├^^^^^^^^^^^^^^^^^^^^┤
+   *│ Details            │
+   *│                    │
+   *└────────────────────┘
+   */
+  | 'topbar';
+
+const GrabberColor = encodeURIComponent(theme.gray300);
+const GrabberSVG = `url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16' fill='${GrabberColor}' height='16px' width='16px'%3E%3Ccircle cx='4.73' cy='8' r='1.31'%3E%3C/circle%3E%3Ccircle cx='4.73' cy='1.31' r='1.31'%3E%3C/circle%3E%3Ccircle cx='11.27' cy='8' r='1.31'%3E%3C/circle%3E%3Ccircle cx='11.27' cy='1.31' r='1.31'%3E%3C/circle%3E%3Ccircle cx='4.73' cy='14.69' r='1.31'%3E%3C/circle%3E%3Ccircle cx='11.27' cy='14.69' r='1.31'%3E%3C/circle%3E%3C/svg%3E")`;
+
+const SIDEBAR_MIN_WIDTH = 325;
+const TOPBAR_MIN_HEIGHT = 325;
+
+type Props = {
+  layout: Layout;
+  showCrumbs?: boolean;
+  showTimeline?: boolean;
+  showVideo?: boolean;
```

Comment:
These are 'extra' features to be sure.
I think if we can figure out how to hide the main sentry sidebar, these would be valuable in making a pop-out inspector window.
It's cheap to keep or remove them.